vars/cloudfront.var.yml

```diff
@@ -11,9 +11,9 @@ cloudformation_stacks:
     template_jinja: "{{ template }}.j2"
     template_parameters:
       OriginReadTimeout: 180 #service increase requested, we now have 3 min wait available
-      WebOrigin: "{{ Environment }}{{ service_name }}WebElbDNSName"
+      WebOrigin: "{{ service_name_lower }}.{{ RootDomain }}"
```

Comment:
you might as well reference the Parma store here also.

Reply:
Done.